daprdocs/content/en/reference/components-reference/supported-bindings/huawei-obs.md

```diff
@@ -0,0 +1,278 @@
+
+### Get object
+
+To perform a get file operation, invoke the Huawei OBS binding with a `POST` method and the following JSON body:
+
+```json
+{
+  "operation": "get",
+  "metadata": {
+    "key": "my-test-file.txt"
+  }
+}
+```
+
+The metadata parameters are:
+
+- `key` - the name of the object
+
+#### Example
+
+{{< tabs Windows Linux >}}
+
+  {{% codetab %}}
+  ```bash
+  curl -d '{ \"operation\": \"get\", \"metadata\": { \"key\": \"my-test-file.txt\" }}' http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
+  ```
+  {{% /codetab %}}
+
+  {{% codetab %}}
+  ```bash
+  curl -d '{ "operation": "get", "metadata": { "key": "my-test-file.txt" }}' \
+        http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
+  ```
+  {{% /codetab %}}
+
+{{< /tabs >}}
+
+#### Response
+
+The response body contains the value stored in the object.
+
+
+### Delete object
+
+To perform a delete object operation, invoke the Huawei OBS binding with a `POST` method and the following JSON body:
+
+```json
+{
+  "operation": "delete",
+  "metadata": {
+    "key": "my-test-file.txt"
+  }
+}
+```
+
+The metadata parameters are:
+
+- `key` - the name of the object
+
+
+#### Examples
+
+##### Delete object
+
+{{< tabs Windows Linux >}}
+
+  {{% codetab %}}
+  ```bash
+  curl -d '{ \"operation\": \"delete\", \"metadata\": { \"key\": \"my-test-file.txt\" }}' http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
+  ```
+  {{% /codetab %}}
+
+  {{% codetab %}}
+  ```bash
+  curl -d '{ "operation": "delete", "metadata": { "key": "my-test-file.txt" }}' \
+        http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
+  ```
+  {{% /codetab %}}
+
+{{< /tabs >}}
+
+#### Response
+
+An HTTP 204 (No Content) and empty body will be retuned if successful.
+
+
+### List objects
+
+To perform a list object operation, invoke the Huawei OBS binding with a `POST` method and the following JSON body:
+
+```json
+{
+  "operation": "list",
+  "data": {
+    "maxResults": 5,
+    "prefix": "dapr-",
+    "marker": "obstest",
+    "delimiter": "jpg"
+  }
+}
+```
+
+The data parameters are:
+
+- `maxResults` - (optional) sets the maximum number of keys returned in the response. By default the action returns up to 1,000 key names. The response might contain fewer keys but will never contain more.
+- `prefix` - (optional) limits the response to keys that begin with the specified prefix.
+- `marker` - (optional) marker is where you want Huawei OBS to start listing from. Huawei OBS starts listing after this specified key. Marker can be any key in the bucket. The marker value may then be used in a subsequent call to request the next set of list items.
+- `delimiter` - (optional) A delimiter is a character you use to group keys. It returns objects/files with their object key other than that is specified by the delimiter pattern.
+
```

Comment:
Do you want to provide an example here of the `list` command?

Reply:
Sure, I will add an example for this.

Reply:
@msfussell Added example. pls review, thanks.